cmd/snap/cmd_run.go

```diff
@@ -423,6 +430,35 @@ func migrateXauthority(info *snap.Info) (string, error) {
 	return targetPath, nil
 }
 
+func straceCmd() ([]string, error) {
+	current, err := user.Current()
+	if err != nil {
+		return nil, err
+	}
+	sudoPath, err := exec.LookPath("sudo")
+	if err != nil {
+		return nil, fmt.Errorf("cannot use strace without sudo: %s", err)
+	}
+	stracePath, err := exec.LookPath("strace")
+	if err != nil {
+		// Ubuntu Core devices will need strace from the snap
+		cand := filepath.Join(dirs.SnapMountDir, "strace-static", "current", "bin", "strace")
```

Comment:
`strace-static` is a special name, perhaps we can call it `snapd-strace-static`?

Reply:
Yeah, happy to change it